app/main.py

```diff
@@ -1,8 +1,36 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
```

Comment:
Please simplify this method as it is too difficult to read:

- First, create a list comprehension of Person objects based on people data.
- Iterate over people data and check if a person has a wife or husband. If so, get the spouse object from the people dictionary (which was populated earlier) and assign it to the current person as their wife or husband.